core/state/snapshot/difflayer.go

```diff
@@ -394,7 +394,7 @@ func (dl *diffLayer) storage(accountHash, storageHash common.Hash, depth int) ([
 	if storage, ok := dl.storageData[accountHash]; ok {
 		if data, ok := storage[storageHash]; ok {
 			snapshotDirtyStorageHitMeter.Mark(1)
-			snapshotDirtyStorageHitDepthHist.Update(int64(depth))
+			//snapshotDirtyStorageHitDepthHist.Update(int64(depth))
```

Comment:
should we delete the comments here

Reply:
It is commented out because of the performance issue, if we need the metrics in some scenery, we get it back without deleting it.